src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax;
+
+public class Truck extends Machine {

```

Comment:
The `Truck` class should override `doWork()` and `stopWork()` methods from the `Machine` class. 🚚